.github/stale.yml

```diff
@@ -18,3 +18,13 @@ markComment: >
 # Comment to post when closing a stale issue. Set to `false` to disable
 closeComment: >
   This issue has been closed due to it having no activity.
+# Issue/PRs having project assigned will not be marked as stale
+exemptProjects: false
+
+pulls:
+  daysUntilStale: 15
+  daysUntilClose: false
```

Comment:
Does this actually work?

Reply:
at least it's documented :) here https://github.com/probot/stale